Lesson_28_Classes/homework/js/script.js

```diff
@@ -0,0 +1,117 @@
+//1) Реализовать класс, описывающий окружность. В классе должны быть следующие компоненты:
+
+//поле, хранящее радиус окружности;
+//get-свойство, возвращающее радиус окружности;
+//set-свойство, устанавливающее радиус окружности;
+//get-свойство, возвращающее диаметр окружности;
+//метод, вычисляющий площадь окружности;
+//метод, вычисляющий длину окружности.
+//Продемонстрировать работу свойств и методов.
+
+class Circle {
+  constructor(r){
+      this.r=r;
+  }
+  get radius(){
+      return this.r;
+  }
+  set radius(){
+      return this.r;
+  }
+  get diametr(){
+      return this.r*2;
+  }
+  area(){
+      return Math.PI*this.r**2;
+  } 
+  circumference(){
+      return Math.PI*this.r*2;
+  }
+}
+
+let circle=new Circle;
+console.log(circle.radius());
+console.log(circle.diametr());
```

Comment:
это у нас свойства, поэтому скобки не нужны
```suggestion
console.log(circle.radius);
console.log(circle.diametr);
```